cli/app.go

```diff
@@ -29,9 +29,15 @@ const (
 	// TODO: RSDK-6683.
 	quietFlag = "quiet"
 
-	logsFlagErrors = "errors"
-	logsFlagTail   = "tail"
-	logsFlagCount  = "count"
+	logsFlagFormat     = "format"
+	logsFlagOutputFile = "output"
+	logsFlagKeyword    = "keyword"
+	logsFlagLevels     = "levels"
+	logsFlagStartTime  = "start"
+	logsFlagEndTime    = "end"
+	logsFlagErrors     = "errors"
+	logsFlagTail       = "tail"
+	logsFlagCount      = "count"
```

Comment:
it looks like some of these were added in a separate PR already. Would you mind pulling down main so we can get a more up-to-date look at what the diff is precisely?